MathLib/LinAlg/Lis/LisMatrix.h

```diff
@@ -45,6 +45,22 @@ struct SetMatrixSparsity<LisMatrix, SPARSITY_PATTERN>;
 class LisMatrix
 {
 public:
+    /// Matrix type
+    enum class MatrixType : int
```

Comment:
I it used somewhere? Since now you've moved it here, you could also have dropped it entirely.